.github/workflows/cicd.yml

```diff
@@ -0,0 +1,24 @@
+name: CI/CD
+
+on: [push]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - uses: actions/setup-node@v2
```

Comment:
These are both on stable v4, might make sense to try to upgrade to the latest if there are no breaking changes?

Reply:
Good idea! Opened a PR here: https://github.com/knocklabs/knock-node/pull/51